play/index.html

```diff
@@ -53,6 +53,20 @@
         <script>
             {{{ script }}}
         </script>
+
+        <!-- LogRocket -->
+        {{#hasLogrocketSession}}
+            <script src="https://cdn.lr-in-prod.com/LogRocket.min.js" crossorigin="anonymous"></script>
+            <script>window.LogRocket && window.LogRocket.init('kocydj/workadventure');</script>
```

Comment:
We should make the key configurable via an environment variable.
I don't particularly want to receive logs from the self-hosted installs :D